### PR TITLE
JBPM-6828 - REST endpoint containers/{containerId}/processes/instance…

### PR DESCRIPTION
**Pull request by tsurdilo**
…s/{instanceId}/signals doesn't list boundary signals

### CODE REVIEW

**Comment by tsurdilo** [comment]
depends on https://github.com/kiegroup/jbpm/pull/1134

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/ProcessServiceIntegrationTest.java [hunk 1]
```diff
@@ -247,6 +247,38 @@ public void testSignalProcessInstanceNullEvent() throws Exception {
...
+    @Test
+    public void testBoundarySignalProcessInstance() throws Exception {
+        Long processInstanceId = processClient.startProcess(CONTAINER_ID, PROCESS_ID_BOUNDARY_SIGNAL_PROCESS);
+        assertNotNull(processInstanceId);
+        assertTrue(processInstanceId.longValue() > 0);
+
+        try {
+            checkAvailableBoundarySignals(CONTAINER_ID, processInstanceId);
+        } catch (Exception e){
+            processClient.abortProcessInstance(CONTAINER_ID, processInstanceId);
```
Comment (sutaakar) [inline kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/ProcessServiceIntegrationTest.java]: Please move abortProcessInstance into the final statement, it should be done in any case.
Also please delete e.printStackTrace();  - it doesn't bring any value.
Also please delete fail(e.getMessage()); - unexpected exceptions should be thrown out of test method, JUnit will handle it as a failing test, printing exception with stacktrace to the log.

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/ProcessServiceIntegrationTest.java [hunk 2]
```diff
@@ -247,6 +247,38 @@ public void testSignalProcessInstanceNullEvent() throws Exception {
...
+    @Test
+    public void testBoundarySignalWithExpressionProcessInstance() throws Exception {
+        Long processInstanceId = processClient.startProcess(CONTAINER_ID, PROCESS_ID_BOUNDARY_SIGNAL_EXPRESSION_PROCESS);
+        assertNotNull(processInstanceId);
+        assertTrue(processInstanceId.longValue() > 0);
+
+        try {
+            checkAvailableBoundarySignals(CONTAINER_ID, processInstanceId);
+        } catch (Exception e){
+            processClient.abortProcessInstance(CONTAINER_ID, processInstanceId);
```
Comment (sutaakar) [inline kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/ProcessServiceIntegrationTest.java]: Same as above.

**Comment by tsurdilo** [comment]
@sutaakar thanks - amended. there were other existing tests doing the same. think i got them all changed now.

**Pull request merged**